tests/Fixer/Basic/OctalNotationFixerTest.php

```diff
@@ -76,8 +76,13 @@ public static function provideFixCases(): iterable
         ];
 
         yield [
-            '<?php $foo = 0o0_123_456;',
-            '<?php $foo = 0_123_456;',
+            '<?php $foo = +0o1_234;',
+            '<?php $foo = +01_234;',
+        ];
+
+        yield [
+            '<?php $foo = -0o123_456;',
```

Comment:
why we changed from `0o0_123` to `0o123` ?

Reply:
Because the fixer should not be responsible for it, `0x00` is unfixed as well.